devicetypes/timevalve-gaslock-t-08/timevalve-smart.src/timevalve-smart.groovy

```diff
@@ -0,0 +1,281 @@
+metadata {
+        definition (name: "Timevalve Smart", namespace: "timevalve.gaslock.t-08", author: "ruinnel") {
+                capability "Valve"
+                capability "Refresh"
+                capability "Battery"
+                capability "Notification"
+				capability "Configuration"
+                capability "Temperature Measurement"
+                
+                command "setRemaining"
+                command "setTimeout"
+                command "setTimeout10"
+                command "setTimeout20"
+                command "setTimeout30"
+                command "setTimeout40"
+                
+                command "remainingLevel"
+                
+                attribute "remaining", "number"
+                attribute "remainingText", "String"
+                attribute "timeout", "number"
+        }
```

Comment:
To clarify - with no fingerprint, this Device Type Handler will never be assigned to the valve when it's paired.